src/api/controllers/authController.js

```diff
@@ -33,6 +34,14 @@ export default class Auth {
       email,
       password: hashedPassword
     });
+    await mailer('Please verify your email', 'Email verification', email, 'notifications', {
+      email,
+      buttonText: 'Verify',
+      userName: username,
+      message: 'Please click on the link to verify your email for authors haven,If you didn\'t request this, simply ignore this e-mail.',
+      link: `${env.baseUrl}/auth/verifyEmail`
+
+    });
     const tokenData = { id: user.dataValues.id, username, email };
```

Comment:
change  auth to users for to reduce confusion since login is using /users not /auth